#Parameters for connecting with virtual robot at Roboshop
robot_ip: "192.168.56.2"  # this the the IP of the Robot
state_server_port: "1104"   # the port of TCP state server
motion_server_port: "1105"    # the port of TCP motion server